mint/tranformedData.ts

```diff
@@ -0,0 +1,15 @@
+import {Dictionary} from "vuex";
+import {evaluate} from "mathjs";
+
+export interface TransformationProps {
+    settings: Dictionary<string>
+}
+
+export function useTransformation(props: TransformationProps) {
+    const evaluateFormula = (formula: string) => {
+        const interpolatedFormula = formula.replace(/\{([^}]+)\}/g,
+            (match) => props.settings[match.replace(/\{|\}/g, "")]);
+       return Math.round(evaluate(interpolatedFormula));
```

Comment:
We might want to add `y_accuracy` as another metadata option so the mintr can also control precision as well.

Reply:
Might just be able to do that through plotly config?

Reply:
The shiny app had an error range for costs too, is that not to be included?

Reply:
No, just has errors for cases averted/efficacy, not for costs